transcom/mymove
# B 18645 move grade to orders from allowances int

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a904659)

## Summary

As part of previous work on the customer view, we moved the Grade and Current Duty Location fields to be associated with the Orders for all customer facing pages.

For the TOO/Services Counselor view, the pay grade is currently displayed under the Allowances section and edited by clicking the "Edit Allowances" button.

This work will be moving the pay grade to be under the Orders section rather than the Allowances section, as well as editing the pay grade by using the "View and edit orders" button.

Changes to be made for TOO/Services Counselor screens:

- Update the pay grade to be shown under the Orders section.

- Remove the pay grade from the Allowances section.

- Update the "View and edit orders" button to allow editing of the pay grade.

- Remove the editing of the pay grade from the "Edit Allowances" section.

### How to test

TOO
1. Access the office app as a TOO
2. Make sure pay grade is under the Orders section
3. Make sure you can edit the pay grade
4. When you edit the pay grade, check that the Allowances -> Weight allowance was updated

Services counselor
1. Access the office app as a TOO
2. Make sure pay grade is under the Orders section
3. Make sure you can edit the pay grade
4. When you edit the pay grade, check that the Allowances -> Weight allowance was updated

## Screenshots
Previous view:
<img width="1078" alt="image" src="https://github.com/transcom/mymove/assets/147537467/bfb111e0-4330-491e-be6a-e6c1ad5fe9f7">

Current view:
<img width="961" alt="image" src="https://github.com/transcom/mymove/assets/147537467/c55be2c7-fc08-4162-904f-e9fecaf79cb5">
